api/document/serializers/content.py

```diff
@@ -123,15 +141,15 @@ def to_representation(self,
     def to_internal_value(self,
                           data: List[PrimitiveDict]) -> List[PrimitiveDict]:
         if not self.is_leaf_node:
-            serializer = NestedAnnotationSerializer(many=True)
-            return serializer.to_internal_value(data)
```

Comment:
OMG I hate DRF's `many=True` thing. Aside from being confusing to developers, it completely changes the type signature for the method, so mypy thought this code was erroneous, which is _totally understandable_.  I don't think using a list comprehension is particularly less efficient or readable, so I just changed it to that, to appease mypy, and maybe even make it less confusing for newbie devs too.